src/main/java/org/csanchez/jenkins/plugins/kubernetes/PodTemplate.java

```diff
@@ -584,6 +588,22 @@ protected Object readResolve() {
         return this;
     }
 
+    @Nonnull
+    public List<Affinity> getAffinities() {
+        if(affinities ==  null) {
```

Comment:
can you fix the formatting generally to match the rest of the code? 
default eclipse java format